src/libs/external-api/api.ts

```diff
@@ -0,0 +1,43 @@
+import { CallbackRateLimiter } from './callback-rate-limiter'
+
+/**
+ * Current version of the Cockpit Widget API
+ */
+export const COCKPIT_WIDGET_API_VERSION = '0.0.0'
```

Comment:
This should be injected in build time, right?

Reply:
yes. where should the "source of truth" live, though?

Reply:
Difficult.
If it was in its own repo, we could track the git release, but in this case we maybe have to hardcode it in the packages.json?